packages/core/src/helpers/chunkify.js

```diff
@@ -0,0 +1,13 @@
+import Loading from "$app/components/Loading";
+import loadable from "@loadable/component";
+
+/**
+ * @param {Promise} statement The import() statement for the given component. Example: import("@datawheel/canon-cms")
+ * @param {String} [name = "default"] The component name to import, needed for named exports.
+ */
+export default (statement, name = "default") => {
+  const resolveComponent = components => components[name];
+  const lazy = loadable(() => statement, {fallback: Loading, resolveComponent});
+  lazy.resolveComponent = resolveComponent;
+  return lazy;
+};
```

Comment:
In the `chunkify` function, the first parameter is the promise returned by the `import()` call. This promise is then wrapped as the result of a function that is called later, but with a value corresponding to the promise's result, which was calculated at some point much earlier.
From a runtime perspective, it makes no difference to call this with `chunkify(() => import("..."))` and replacing `() => statement` to `statement`, but this would allow the promise to be executed until it's really needed.

Reply:
Fixed this. The `chunkify` function now accepts the anonymous function as its first argument. Even on just the homepage, this cut the lighthouse Time to Interactive and Total Blocking Time scores in half!